src/test/java/com/faforever/client/remote/ServerAccessorImplTest.java

```diff
@@ -206,6 +206,8 @@ public void testConnectAndLogIn() throws Exception {
     assertThat(result.getMessageType(), is(FafServerMessageType.WELCOME));
     assertThat(result.getId(), is(playerUid));
     assertThat(result.getLogin(), is(username));
+
+    instance.disconnect();
```

Comment:
I don't know what this does (especially in a test), but when you expect it to do something you should assert or verify it. (Also applies to all following calls)

Reply:
This just cleans up after the test, not really part of the test itself (if missing, the teardown method killing the sockets will cause an EOFException). And I cant put it in teardown because its only needed in some tests (those that connect at the beginning).